src/local/apps/bl-tool/commands/GenerateBase64Resource.h

```diff
@@ -0,0 +1,230 @@
+/*
+ * This file is part of the swblocks-baselib library.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __APPS_BLTOOL_COMMANDS_GENERATEBASE64RESOURCE_H_
+#define __APPS_BLTOOL_COMMANDS_GENERATEBASE64RESOURCE_H_
+
+#include <apps/bl-tool/GlobalOptions.h>
+
+#include <baselib/cmdline/CommandBase.h>
+
+#include <baselib/core/FileEncoding.h>
+#include <baselib/core/SerializationUtils.h>
+#include <baselib/core/StringTemplateResolver.h>
+#include <baselib/core/StringUtils.h>
+#include <baselib/cmdline/CommandBase.h>
+
+namespace
+{
+    const char* resourceTemplate =
+        "/*\n"
```

Comment:
you can use raw string literal `R"(...)"` to avoid some quotes and explicit \n

Reply:
Kalin and I discussed offline